controller/src/main/java/org/jboss/as/controller/AttributeDefinition.java

```diff
@@ -81,7 +81,7 @@
     private final ParameterCorrector valueCorrector;
     private final ParameterValidator validator;
     private final EnumSet<AttributeAccess.Flag> flags;
-    protected final AttributeMarshaller attributeMarshaller;
+    protected final AttributeMarshaller marshaller;
```

Comment:
This is a breaking change, as the field is visible.

Let's 

private final AttributeMarshaller marshaller;
/** @deprecated use {@link #getMarshaller()} */
@Deprecated
protected final AttributeMarshaller attributeMarshaller;

and in c'tor

this.attributeMarshaller = marshaller;

protected fields are annoying because it's too easy to break them.